go/pkg/apis/recognizer/component_recognizer.go

```diff
@@ -82,14 +83,19 @@ func DetectComponentsWithSettings(settings model.DetectionSettings) ([]model.Com
 }
 
 func detectComponentsWithSettings(settings model.DetectionSettings, ctx *context.Context) ([]model.Component, error) {
+	alizerLogger := utils.GetOrCreateLogger()
+	alizerLogger.V(0).Info("starting component with settings detection")
```

Comment:
Logs aren't combined with other messages so I think we should use capital letters at the beginning of the message

Reply:
Done